### ISSUE TITLE
TV mode

### ISSUE THREAD

**Issue by ranbena** [getredash/redash#117]
Have a fullscreen button on dashboards (would be cool to use the [fullscreen API](https://developer.mozilla.org/en-US/docs/Web/Guide/API/DOM/Using_full_screen_mode))

Perhaps use reveal.js to navigate between dashboard items, but if it's not supported by AngularJS you can always [do it yourself](http://desandro.github.io/3dtransforms/docs/carousel.html).

**Comment by shenyubao** [comment]
:+1:

**Comment by TonyStarkBy** [comment]
So what about fullscreen? There is a cool library - https://github.com/sindresorhus/screenfull.js
Only one small issue in safari: keyboard is not working in fullscreen mode, but it's not need :)

P. S. https://sindresorhus.com/screenfull.js you can test it

**Comment by rohanpd** [comment]
This issue is solved now isn't it? by #748

**Comment by arikfr** [comment]
@rohanpd not really, because if you have more widgets than fits the screen it's not really helping. Proper TV mode will probably have some auto scrolling support.
